src/main/java/gmailgdogra/Officer.java

```diff
@@ -0,0 +1,73 @@
+package gmailgdogra;
+
+public class Officer {
+
+    private String firstName;
```

Comment:
It's unlikely that any officer will change their name during their shift, so I guess you may make these fields final

Reply:
made firstName and lastName final.